__tests__/app.test.js

```diff
@@ -12,20 +13,32 @@ afterAll(() => {
 });
 
 
-  describe("/api/anything", () => {
+describe("/api", ()=>{
+    test("Get: 200 - responds with all the endpoints", ()=>{
+        return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({ body }) => {
+            expect(typeof body.endpoints).toBe("object");
+            expect(body.endpoints).toEqual(endpoints);
+        });
+    })
+})
+
+describe("/api/anything", () => {
```

Comment:
Let's change this describe block description to some thing more descriptive - perhaps something like "General error handling" ?